protostar/commands/test/test_cases.py

```diff
@@ -24,8 +25,7 @@ def display(self, include_stdout_section: bool = False) -> str:
 class PassedTestCase(TestCaseResult):
     test_case_name: str
     execution_resources: Optional[ExecutionResourcesSummary]
-    captured_setup_stdout: str
-    captured_test_stdout: str
+    captured_stdout: List[Tuple[OutputName, str]] = field(default_factory=list)
```

Comment:
```suggestion
    captured_stdout: Dict[OutputName, str] = field(default_factory=dict)
```

we have ordered dicts by default in Python 3.7